WebApplication/Controllers/ProjectsController.cs

```diff
@@ -79,7 +81,7 @@ public async Task<IEnumerable<ProjectDTO>> ListAsync()
         }
 
         [HttpPost]
-        public async Task<ActionResult<ProjectDTO>> CreateProject([FromForm]NewProjectModel projectModel)
+        public async Task<ActionResult<string>> CreateProject([FromForm]NewProjectModel projectModel)
```

Comment:
Should not it cause changes on client-side? (or maybe it's not used at all?)

Reply:
ProjectDTO was mistake, it was always returning string. Just mistakenly mentioned in function definition that it returns ProjectDTO

Reply:
Actually I have just fixed the method signature. The type of returned payload was string prior my changes.